### ISSUE TITLE
Can't reach unique pins

### ISSUE THREAD

**Issue by kristiyandobrev** [bigfish24/ABFRealmMapView#29]
Consider that I am having the following code:

```
func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
        if let annotationABF = annotation as? ABFAnnotation {
            if annotationABF.type == .unique {
            ......
            //more code
```

I am trying to get to the unique pins, but it seems like the I can never reach inside the ` if annotationABF.type == .unique `. Quite strange because  `annotationABF.type` of all the objects on the map seems to return `.cluster`.

A workaround that I found was having:

```
if annotationABF.safeObjects.count == 1 {
  guard let object = annotationABF.safeObjects.first?.rlmObject() as? MyObject else {
                    return nil
    }
}
```

 So Any ideas of why am I not getting to the unique pins using `annotationABF.type == .unique`  ?

**Comment by ajandrade** [comment]
@kristiyandobrev Any update on this?
I think even the annotations with only one element are being considered as .cluster.

And the count label are not right too:

<img width="303" alt="screen shot 2017-03-15 at 23 13 53" src="https://cloud.githubusercontent.com/assets/25066243/23974808/2d3313ce-09d5-11e7-94c0-fe3838cdf14b.png">

Maybe a problem when reusing annotations @bigfish24 ?

**Comment by kristiyandobrev** [comment]
Well, still haven't got any different solution then the workaround I show.

**Comment by bigfish24** [comment]
ah! that's a small bug, where it should check to see if the cluster is 1 and set to unique. I will fix, thanks for reporting!

**Comment by bigfish24** [comment]
Fixed in v2.1.2:
```swift
func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
    if let annotationABF = annotation as? Annotation {
        if annotationABF.type == .unique {
            // ...
        }
    }
    // ... 
}
```
